spec/support/saml_response_doc.rb

```diff
@@ -34,7 +34,7 @@ def input_id
   def raw_xml_response
     if @test_type == 'feature'
       xml_response
-    elsif @response.body =~ /<html>/
+    elsif @response.body.match?(/<html>/)
```

Comment:
These `match?` changes seem to be unrelated Rubocop fixes. Can we do these in a separate PR?